styles/nav-bar.css

```diff
@@ -70,6 +71,34 @@
   margin-right: 20px;
 }
 
+.nav-toggle-show {
+  visibility: visible;
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  top: 0;
+  left: 0;
+  background: #fff;
+  width: 100%;
+  height: 100vh;
+  z-index: 1;
+  transition: all 0.5s ease-in-out;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-bottom-left-radius: 50px;
+  border-bottom-right-radius: 50px;
+  justify-content: flex-start;
+  padding-left: 1rem;
+  padding-top: 2rem;
+  gap: 20px;
+}
+
+.nav-item-show {
+  visibility: visible;
+  width: 100%;
+  justify-content: center;
+  margin-top: 2rem;
+}
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the menu follows the design [template 3](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468) as provided. The font sizes need to be increased for it to look more like the template design. Please consider removing the "gap: 20px" in between the nav items for it to meet up with the spacing as in the template design. I removed the gap and it worked perfectly.
 
 Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="372" alt="Screenshot 2023-05-17 at 18 44 07" src="https://github.com/waiyankyaw961999/Microverse-portofolio/assets/58692590/137076d7-f843-423f-a2d3-a5f97c35d81b"> | <img width="314" alt="Screenshot 2023-05-17 at 18 34 11" src="https://github.com/waiyankyaw961999/Microverse-portofolio/assets/58692590/b51bc630-50d0-4542-a5a2-c12a5a4b0249">